apps/vth-frontend/src/styles/globals.css

```diff
@@ -153,6 +153,35 @@ a {
   max-width: var(--utrecht-page-content-max-width);
 }
 
+@media screen and (-ms-high-contrast: active), screen and (forced-colors: active) {
+  .utrecht-accordion__header .utrecht-accordion__button {
+    border: 2px solid white;
```

Comment:
`white` ➝ `currentColor`

Reply:
Gedaan!
P.S. currentColor is nu currentcolor

Reply:
@Robbert 
Ik krijg van de Linter dat ik moet `currentColor` hebben in de code in plaats van `currentcolor`.

[Maar op de MDN site staat er dat nu `currentcolor` goed is](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentcolor_keyword)

Mag ik de Linter veranderen zodat het `currentcolor` goed vindt?

<img width="707" alt="Screenshot 2024-01-29 at 15 21 31" src="https://github.com/frameless/strapi/assets/11764984/238b4c9b-b0e9-4e43-b8df-02ee1611c788">